### ISSUE TITLE
Support Python3 ?

### ISSUE THREAD

**Issue by peritus** [xyb/robotframework-debuglibrary#20]
Trying to use this library using Python 3 and this happens:

```
> Element Should Be Enabled  accessibility_id=Foo
< keyword: Element Should Be Enabled  accessibility_id=Foo
! FAILED: AttributeError("'str' object has no attribute 'decode'",)
```

I can haz Python3 support ?

**Comment by xyb** [comment]
@peritus I think #21 have resolved this, you may like to upgrade robotframework-debuglibrary to version 2.8.1
